debian/ceph-mgr.postinst

```diff
@@ -24,6 +24,13 @@ set -e
 
 case "$1" in
     configure)
+	CERT="/etc/ceph/ceph-mgr-restful.crt"
+	PKEY="/etc/ceph/ceph-mgr-restful.key"
+	if [ ! -e "$CERT" -o ! -e "$PKEY" ]; then
+	    openssl req -new -nodes -x509 \
+		-subj "/C=US/ST=Oregon/L=Portland/O=IT/CN=ceph-mgr-restful" \
```

Comment:
I was not sure whether those are mandatory or not but my testing suggests we can left them out.